src/libstd/fs.rs

```diff
@@ -114,6 +114,11 @@ pub struct Metadata(fs_imp::FileAttr);
 /// information like the entry's path and possibly other metadata can be
 /// learned.
 ///
+/// #### Note: Iteration Order is Implementation-Defined
```

Comment:
I would remove this; it feels redundant with the more specific sentence below. Moreover, "implementation defined" means that each implementation must define the semantics and this feels more "unspecified".

Reply:
I guess if we are removing one (line 117) makes sense to also remove the other.

Reply:
haha, that was implied ;)